string object_name
geometry_msgs/Pose place_pose
---
bool result